contrib/standalone/qasm_simulator.cpp

```diff
@@ -95,6 +114,7 @@ int main(int argc, char **argv) {
     // Initialize simulator
     AER::Simulator::QasmController sim;
     // Check for config
+    sim.set_config(config);
```

Comment:
Thanks. Maybe we should prioritize command line option rather than qobj. 
```
    json_t config_all = qobj["config"];
    config_all.update(config.begin(), config.end());
    sim.set_config(config_all);
```

Reply:
Agree!